app/pulp/app/pulp_manager.py

```diff
@@ -0,0 +1,14 @@
+"""This module provides system wide command to interface with the pulp platform.
+
+In future commands that can be issued should be limitted this will probably be achieved
+by implementing own ManagmentUtility similar to this one:
+https://github.com/django/django/blob/stable/1.8.x/django/core/management/__init__.py
```

Comment:
Docstrings should talk about the present, not the future. If we want to implement something in the future, we should track that in Redmine, so this docstring should only be the first line.